### PR TITLE
Splitting conferences year wise

### LINKED ISSUE #28: Make the list yearwise
The page is getting longer day to day.
How about listing the conferences by year
For ex: We mention the year in the readme and a link which redirects to the conferences of that year.

### PR DESCRIPTION
**Pull request by techsethi**
Please check & merge. Thanks.

Fixes #28.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,88 +6,21 @@ To add a conference to the list, please [contribute](#contribute)!
...
-## 2018
+## 2017
 
-| Conference | Date | Venue | Description |
-|------------|------|-------|-------------|
-| [Great Indian Developer Summit](http://www.developermarch.com/developersummit/index.html) | 24-27 Apr | Bengaluru | 	With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem.
-| [GopherCon India](http://www.gophercon.in/) | 09-10 Mar | Sheraton Grand, Sangamvadi, Pune | Go conference in India. Go is an open source project developed by a team at Google. |
-| [PyCon Pune](https://pune.pycon.org/2018/) | 08-11 Feb | Pune | The community will meet for two days of talks and work on upstream projects in two days of dev sprint. |
-| [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on hosting, domain, blogging and web technologies. |
+[List of Conferences](year-pages/2017-conferences-list.md)
```
Comment (nikhita) [inline README.md]: I'd like to keep the current year as-is in the main README and not move it to a separate directory.

Can you move the 2017 conferences back to the README? Thanks!

Comment (nikhita) [reply README.md]: So that people can view the current year's list without any further clicks. Most of the time, they just want to view the current year.

Comment (techsethi) [reply README.md]: Yeah, that makes sense abt 2017 being in the main page itself.

am not clear about squashing commit into a single commit.

Comment (nikhita) [reply README.md]: @techsethi That's all right then, I'll use the Github UI to squash the commits.

Please make the changes for the 2017 one, I'll take care of the squashing. :smile:

File: README.md [hunk 2]
```diff
@@ -6,16 +6,9 @@ To add a conference to the list, please [contribute](#contribute)!
 
 If you would like to be notified of future conferences that would be added to the list, you could watch this repo. New additions are made via PRs and have full description of the conference so that you can view it directly in your email. ([Example PR](https://github.com/nikhita/tech-conferences-india/pull/25))
 
-## 2018
+## 2017 (Current Year)
```
Comment (nikhita) [inline README.md]: Just 2017 will be nice.

File: README.md [hunk 3]
```diff
@@ -59,35 +52,25 @@ If you would like to be notified of future conferences that would be added to th
...
-| [RubyConf India](http://rubyconfindia.org/rci2016/) | 19-20 Mar | Le Meridien, Kochi |  RubyConf India is a global event complementing other RubyConf events across the world. |
-| [PyDelhi Conf](https://conference.pydelhi.org/2016/) | 5 Mar | Jawaharlal Nehru University, Delhi | PyDelhi conference is hosted annually by PyDelhi Community with an aim to promote Python programming language. |
-| [FOSSMeet](http://fossmeet.in/2016/) | 26-28 Feb | NIT Calicut | FOSSMeet is an annual event on Free and Open source software, conducted at National Institute of Technology, Calicut. |
-| [GopherCon India](http://www.gophercon.in/) | 19-20 Feb | Vivanta by Taj, Bengaluru | Go conference in India. Go is an open source project developed by a team at Google and many contributors from the open source community. |
-|[Mini Deb-Con](http://in2015.mini.debconf.org/)|30 - 31 Jan| Andheri (E), Mumbai | It is an awesome opportunity for any user who wants to experience the magic of FOSS |
+### 2018
```
Comment (nikhita) [inline README.md]: As a bullet point: `+ 2018 - [List of Conferences](year-pages/2018-conferences-list.md)` in one line.

**Comment by techsethi** [comment]
@nikhita Done, pls check.

File: README.md [hunk 4]
```diff
@@ -59,35 +52,25 @@ If you would like to be notified of future conferences that would be added to th
...
+
+[List of Conferences](year-pages/2018-conferences-list.md)
+
+## Past Years
+
+### 2016
```
Comment (nikhita) [inline README.md]: As a bullet point: `+ 2016 - [List of Conferences](year-pages/2016-conferences-list.md)` in one line.

File: README.md (hunk 2 above)
Comment (techsethi) [reply README.md]: ok. done.

File: README.md (hunk 4 above)
Comment (techsethi) [reply README.md]: ok, done.

**Comment by techsethi** [comment]
done more changes. please check.

**Comment by nikhita** [comment]
@techsethi It looks good, why did you close the PR? :)

**Comment by techsethi** [comment]
Great. Thanks.

**Pull request merged**